deploy/crds.yml

```diff
@@ -551,6 +555,10 @@ spec:
                 description: Name of the account to which the Consumer belongs.
                 type: string
                 pattern: '^[^.*>]*$'
+              preventDelete:
+                description: Indicates if a Consumer can be deleted (true) or not (false)
```

Comment:
This should say `When true, the managed Consumer will not be deleted when the resource is deleted`

Reply:
I made this change to the description